labs/bbb-webrtc-sfu/config/default.example.yml

```diff
@@ -18,6 +18,10 @@ to-akka: "to-akka-apps-redis-channel"
 webcam-force-h264: true
 screenshare-force-h264: true
 
+recordScreenSharing: true
+recordWebcams: true
```

Comment:
We'll be leaving them true by default?

Reply:
It's the example file, but I think you're right.... we can probably leave them disabled by default

Reply:
Yes, this should be true by default - but the sfu should not record video to files for meetings that do not have recording enabled